docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -1065,138 +1241,133 @@ Increment variable assigned by [assignVar](#assignVar) by 1.
-|[forOwn](https://github.com/helpers/handlebars-helpers#forOwn)|
-|[toPath](https://github.com/helpers/handlebars-helpers#toPath)|
-|[get](https://github.com/helpers/handlebars-helpers#get)|
-|[getObject](https://github.com/helpers/handlebars-helpers#getObject)|
-|[hasOwn](https://github.com/helpers/handlebars-helpers#hasOwn)|
-|[isObject](https://github.com/helpers/handlebars-helpers#isObject)|
-|[merge](https://github.com/helpers/handlebars-helpers#merge)|
-|[JSONparse](https://github.com/helpers/handlebars-helpers#JSONparse)|
-|[JSONstringify](https://github.com/helpers/handlebars-helpers#JSONstringify)|
-|[camelcase](https://github.com/helpers/handlebars-helpers#camelcase)|
-|[capitalize](https://github.com/helpers/handlebars-helpers#capitalize)|
-|[capitalizeAll](https://github.com/helpers/handlebars-helpers#capitalizeAll)|
-|[center](https://github.com/helpers/handlebars-helpers#center)|
-|[chop](https://github.com/helpers/handlebars-helpers#chop)|
-|[dashcase](https://github.com/helpers/handlebars-helpers#dashcase)|
-|[dotcase](https://github.com/helpers/handlebars-helpers#dotcase)|
-|[hyphenate](https://github.com/helpers/handlebars-helpers#hyphenate)|
-|[isString](https://github.com/helpers/handlebars-helpers#isString)|
-|[lowercase](https://github.com/helpers/handlebars-helpers#lowercase)|
-|[occurrences](https://github.com/helpers/handlebars-helpers#occurrences)|
-|[pascalcase](https://github.com/helpers/handlebars-helpers#pascalcase)|
-|[pathcase](https://github.com/helpers/handlebars-helpers#pathcase)|
-|[plusify](https://github.com/helpers/handlebars-helpers#plusify)|
-|[reverse](https://github.com/helpers/handlebars-helpers#reverse)|
-|[sentence](https://github.com/helpers/handlebars-helpers#sentence)|
-|[snakecase](https://github.com/helpers/handlebars-helpers#snakecase)|
-|[split](https://github.com/helpers/handlebars-helpers#split)|
-|[startsWith](https://github.com/helpers/handlebars-helpers#startsWith)|
-|[titleize](https://github.com/helpers/handlebars-helpers#titleize)|
-|[trim](https://github.com/helpers/handlebars-helpers#trim)|
-|[uppercase](https://github.com/helpers/handlebars-helpers#uppercase)|
-|[encodeURI](https://github.com/helpers/handlebars-helpers#encodeURI)|
-|[decodeURI](https://github.com/helpers/handlebars-helpers#decodeURI)|
-|[urlResolve](https://github.com/helpers/handlebars-helpers#urlResolve)|
-|[urlParse](https://github.com/helpers/handlebars-helpers#urlParse)|
-|[stripProtocol](https://github.com/helpers/handlebars-helpers#stripProtocol)|
-|[enumerate](https://github.com/helpers/handlebars-helpers#enumerate)|
-|[equals](https://github.com/helpers/handlebars-helpers#equals)|
-|[getShortMonth](https://github.com/helpers/handlebars-helpers#getShortMonth)|
-|[pick](https://github.com/helpers/handlebars-helpers#pick)|
+The following standard helpers are available to all Stencil themes:
+
+* [after](https://github.com/helpers/handlebars-helpers#after)

```

Comment:
I wonder if we can break them up into groups so it is easier to navigate for folks. I am also wondering if we can add a 2 line description or example to make it easier and keep them on our docs vs them jumping back and forth between the 2.